doc/releases/luminous.rst

```diff
@@ -1,3 +1,286 @@
+v12.2.7 Luminous
+================
+
+This is the seventh bugfix release of Luminous v12.2.x long term
+stable release series. This release contains severalx fixes for
+regressions in the v12.2.6 and v12.2.5 releases.  We recommand that
+all users upgrade.
+
+:note: The v12.2.6 release has serious known regressions.  If you installed this release, please see the upgrade procedure below.
+
+:note: The v12.2.5 release has a potential data corruption issue with erasure coded pools.  If you ran v12.2.5 with erasure coding, please see below.
+
+       
+Upgrading from v12.2.6
+----------------------
+
+v12.2.6 included an incomplete backport of an optimization for
+BlueStore OSDs that avoids maintaining both the per-object checksum
+and the internal BlueStore checksum.  A critical follow-on patch was
+not included, which introduced a bug that corrupted (failed to update)
```

Comment:
Due to the accidental omission of a critical follow-on patch, v12.2.6 corrupts (fails to update)